src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/images/by-condition.svg

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
I think these SVG images area great improvement over what we had before. Nice work!